src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader{
+    @Override
+    public List<String> read(String file) {
+        try {
+            return Files.readAllLines(Path.of(file));
+        } catch (IOException e) {
+            throw new RuntimeException("There was an error reading file.", e);
+        }
+    }
```

Comment:
The class 'FileReaderImpl' should be placed in a package that reflects its purpose. Consider placing it in a 'service' or 'service.impl' package.